cfavml/src/danger/impl_neon.rs

```diff
@@ -146,6 +146,50 @@ impl SimdRegister<f32> for Neon {
         vst1q_f32(mem, reg)
     }
 }
+const EXPONENT_MASK_F32: u32 = 2139095040;
+const MANTISSA_MASK_F32: u32 = 8388607;
+impl Hypot<f32> for Neon {
+    #[inline(always)]
+    unsafe fn hypot(x: Self::Register, y: Self::Register) -> Self::Register {
+        // Convert inputs to absolute values
+        let (x, y) = (vabsq_f32(x), vabsq_f32(y));
+
+        // Find the max and min of the two inputs
+        let (hi, lo) = (vmaxq_f32(x, y), vminq_f32(x, y));
```

Comment:
Same vibe as mentioned earlier, best to have these just as two separate lines rather than a tuple.

Reply:
It's locally changed, I just kept forgetting to save the files after making the changes